app/jobs/expire_token_job.rb

```diff
@@ -0,0 +1,9 @@
+class ExpireTokenJob < ApplicationJob
+  queue_as :default
+
+  def perform(params = {})
```

Comment:
**params

Reply:
`ArgumentError:
       wrong number of arguments (given 1, expected 0)`